Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "discord:brainfood#7460 github: brainfood1183",
```

Comment:
you seem to have left your previous pr's commits bleed into this one. These need to be removed.